custom-elements/builtin-coverage/template.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html is="my-html">
+<head>
+<meta charset="utf-8">
+<meta name="help" content="https://html.spec.whatwg.org/multipage/custom-elements.html#element-definition">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  let contentInConstructor = undefined;
+  let numberOfContentChildNodesInConnectedCallback = 0;
+  let MyTemplate = class extends HTMLTemplateElement {
+    constructor(...args) {
+      super(...args);
+      contentInConstructor = this.content;
+    }
+    connectedCallback(...args) {
+      super.connectedCallback && super.connectedCallback();
```

Comment:
This is dead code because HTMLTemplateElement will never have connectedCallback. Remove?